ci/run.sh

```diff
@@ -122,4 +122,5 @@ else
     $cmd --no-default-features
     $cmd
     $cmd --features extra_traits
+    $cmd --all-features
```

Comment:
Ah, I see this answers my question. We already run checks with `extra_traits` and `rustc-dep-of-std` independently here https://github.com/rust-lang/libc/blob/149d3922e3493f14ec03f0bdd7968e0fe85b5ddf/ci/verify-build.sh#L64-L75, making them work together isn't a concern. So you can drop this part of the change and anything that was added to fix it.